Fix GoToEvent link generation when value is None

hramezani opened the pull request.

sydney-runkle commented on src/npm-fastui/src/components/display.tsx:
```diff
@@ -218,9 +218,7 @@ const subKeys = (template: string, row: DataModel): string | null => {
-    } else if (v === null) {
+    if (v === null || v === undefined) {
       returnNull = true
       return 'null'
```
Why do we return `'null'` here and `null` below? Do we no longer need to raise an error?

hramezani replied:
> Why do we return 'null' here and null below?

Don't know.

> Do we no longer need to raise an error?

I think we should keep it like that. I will change the implementation

hramezani commented:
Ok, I changed the implementation.

The table renders without any error with my patch. you can see the row begins with `Tokyo` is not rendered as a link (because I changed the data to reproduce the problem and confirm my fix)

<img width="1379" alt="image" src="https://github.com/pydantic/FastUI/assets/3122442/1c0e9174-2390-4e21-aa4e-7840575bbb84">

but it fails without my patch:

<img width="1191" alt="image" src="https://github.com/pydantic/FastUI/assets/3122442/1c7e3929-3991-44a4-95ac-5bd3e3267dfd">

sydney-runkle reviewed (approved):
Looks great, thanks @hramezani
